### ISSUE TITLE
Can't save new characters.

### ISSUE THREAD

**Issue by Sooose** [d3athrow/vgstation13#19611]
Players can't save NEW character slots, they can still edit old ones. This is particularly troublesome for newcomers to the server as they are unable to save any character as they have no slots ready yet. I discovered this through an ahelp from a new player and discovered that I was able to edit my own characters, but not save new slots either. Also, whenever someone tries to save a new character the admin logs pop up with these errors:
```
ADMIN LOG: Error #: 1 - no such column: pulltoggle
ADMIN LOG: InsertPlayer: Error #:1 - 17 values for 18 columns
```

If you edit and save a character slot you already have, you still get the error:
`ADMIN LOG: Error #: 1 - no such column: pulltoggle`

I tried downgrading from 512 to 510.1347 with no luck, thinking it might be a new byond version issue as we were both on 512. Another weird thing to note is that a little box appears on your load character screen when you fail to save it, like so 
![ss 2018-09-06 at 11 30 01](https://user-images.githubusercontent.com/28773503/45168005-56a48c00-b1c8-11e8-863c-6a0844a3154f.png)

**Comment by d3athrow** [comment]
e6764af67499f721deedbd7cc2863d24eb4b0b53
Hopefully this fixes it.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Well as the responsible party I'd like to apologise. I'm still not sure what went wrong.

I just copied what the attack animations PR did in hopes of making a very simple char preferences PR for people to copy but evidently I flubbed one or two things up.

**Comment by d3athrow** [comment]
I really should have looked at your original PR cause i guess what is wrong is the migration file.

**Comment by d3athrow** [comment]
Fixed with 34f72c28e6be4f001e3c6b8bd05d1747097c7671

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Oh that explains it, I probably copied what attack animations did in the code, but copied the flavor text migration file instead

How embarassing
